src/cljs/main/broadfcui/common/codemirror.cljs

```diff
@@ -54,7 +54,7 @@
      (react/call :call-method this "off" event-type listener))
    :call-method
    (fn [{:keys [locals]} method & args]
-     (apply utils/call-external-object-method (aget (:code-mirror-component @locals) "doc") method args))
+     (js-invoke (aget (:code-mirror-component @locals) "doc") method args))
```

Comment:
Actually, I believe this still needs the `apply` in front.  I'm running into the same issue locally now.